operator/pkg/operator.go

```diff
@@ -455,6 +455,18 @@ func (o *Operator) afterHandlingBatchV3(log *servicemanager.ContractAlignedLayer
 }
 
 func (o *Operator) verify(verificationData VerificationData, results chan bool) {
+	disabledVerifiersBitmap, err := o.avsReader.DisabledVerifiers()
```

Comment:
It is getting the verification bitmat for each proof. If the batch is too big, it will add a lot of latency

Reply:
Good catch, fixed it in [dd8ff97](https://github.com/yetanotherco/aligned_layer/pull/1090/commits/dd8ff97a61673c474afd8361470fd4a9d6f7a5d7)